build.sbt

```diff
@@ -1,51 +1,45 @@
+import Versions._
 import BuildHelper._
 
+enablePlugins(EcosystemPlugin)
```

Comment:
Can we automatically enable it?

Also it should be called `ZioEcosystemPlugin` because `EcosystemPlugin` is too ambiguous (which ecosystem?).

Reply:
Done